command args: trim leading/trailing whitespace

ThomasAdam opened the pull request:
When certain commands in tasksh (such as 'review N') contain whitespace,
such as:

    review '1     '

The whitespace would tokenise into the command arg vector, effectively
treating the whitespace as a literal component.

In the case of the 'review' command, this is undesirable.  Therefore,
strip out all leading/trailing whitespace from the command args vector.

Fixes GH issue #43

pbeckingham commented:
This is a good idea, but it's on the master branch (should be on 1.3.0), and I don't know what the WHITESPACE_CHARS define is for.

ThomasAdam commented:
Oops, that was Github being too helpful in setting the base automatically.  Changed now.

As for `WHITESPACE_CHARS` that was a hold-over from a different change.  I've removed it. Thanks for spotting it!

pbeckingham commented:
Nice, thanks.
